medkumo.js

```diff
@@ -125,6 +119,9 @@
         if (!element) {
             element = '.medkumo-sdk-body';
         }
+        if (Medkumo.jQuery(element).length == 0) {
```

Comment:
by default element is '.medkumo-sdk-body', so update default parameter then remove line 119 - 121